### ISSUE TITLE
Master currently broken

### ISSUE THREAD

**Issue by plexus** [plexus/chestnut#106]
I updated all dependencies, unfortunately things now no longer work. There's no obvious clue what's going on, and I need to move on to other stuff today, so logging this here.

```
 lein repl
nREPL server started on port 56708 on host 127.0.0.1 - nrepl://127.0.0.1:56708
REPL-y 0.3.5, nREPL 0.2.6
Clojure 1.6.0
Java HotSpot(TM) 64-Bit Server VM 1.8.0_31-b13
    Docs: (doc function-name-here)
          (find-doc "part-of-name-here")
  Source: (source function-name-here)
 Javadoc: (javadoc java-object-or-class-here)
    Exit: Control+D or (exit) or (quit)
 Results: Stored in vars *1, *2, *3, an exception in *e

sesame-seed.server=> (run)
Figwheel: Starting server at http://localhost:3449
2015-03-15 12:32:31.495:INFO:oejs.Server:jetty-7.6.13.v20130916
2015-03-15 12:32:31.558:INFO:oejs.AbstractConnector:Started SelectChannelConnector@0.0.0.0:10555
Starting web server on port 10555 .
#<Server org.eclipse.jetty.server.Server@495948cc>
sesame-seed.server=> Compiling "resources/public/js/app.js" from ["env/dev/cljs" "src/cljs"]...
Compiling "resources/public/js/app.js" failed.
clojure.lang.ExceptionInfo: failed compiling file:env/dev/cljs/sesame_seed/main.cljs
 at clojure.core$ex_info.invoke (core.clj:4403)
    cljs.compiler$compile_file$fn__4544.invoke (compiler.clj:1130)
    cljs.compiler$compile_file.invoke (compiler.clj:1101)
    cljs.compiler$compile_root.invoke (compiler.clj:1160)
    cljs.closure$compile_dir.invoke (closure.clj:357)
    cljs.closure$eval4910$fn__4911.invoke (closure.clj:397)
    cljs.closure$eval4846$fn__4847$G__4837__4854.invoke (closure.clj:305)
    cljs.closure$eval4897$fn__4898.invoke (closure.clj:411)
    cljs.closure$eval4846$fn__4847$G__4837__4854.invoke (closure.clj:305)
    clojurescript_build.core.CompilableSourcePaths/fn (core.clj:18)
    clojure.core$map$fn__4245.invoke (core.clj:2557)
    clojure.lang.LazySeq.sval (LazySeq.java:40)
    clojure.lang.LazySeq.seq (LazySeq.java:49)
    clojure.lang.RT.seq (RT.java:484)
    clojure.core$seq.invoke (core.clj:133)
    clojure.core$apply.invoke (core.clj:624)
    clojure.core$mapcat.doInvoke (core.clj:2586)
    clojure.lang.RestFn.invoke (RestFn.java:423)
    clojurescript_build.core.CompilableSourcePaths._compile (core.clj:18)
    cljs.closure$build.invoke (closure.clj:1393)
    clojurescript_build.core$build_source_paths_STAR_.invoke (core.clj:150)
    figwheel_sidecar.auto_builder$default_build_options$fn__15612.invoke (auto_builder.clj:66)
    figwheel_sidecar.auto_builder$warning$fn__15609.invoke (auto_builder.clj:61)
    clojurescript_build.auto$time_build$fn__15422.invoke (auto.clj:55)
    clojurescript_build.auto$after$fn__15428.invoke (auto.clj:64)
    clojurescript_build.auto$after$fn__15428.invoke (auto.clj:64)
    clojurescript_build.auto$after$fn__15428.invoke (auto.clj:64)
    clojurescript_build.auto$error$fn__15431.invoke (auto.clj:71)
    clojurescript_build.auto$before$fn__15425.invoke (auto.clj:60)
    clojurescript_build.auto$make_conditional_builder$fn__15443.invoke (auto.clj:103)

sesame-seed.server=> 
```

**Comment by swannodette** [comment]
@plexus the latest version of ClojureScript is 0.0-3117, did you try this?

**Comment by plexus** [comment]
This was on 0.0-3058, but just tried it on 0.0-3117 and same thing. I put a demo app on https://github.com/plexus/sesame-seed

**Comment by plexus** [comment]
Doing a `lein cljsbuild once` works, so might be we're doing something wrong when calling figwheel-sidecar over here: https://github.com/plexus/sesame-seed/blob/master/env/dev/clj/sesame_seed/dev.clj#L24-L33

/cc @bhauman

**Comment by timgilbert** [comment]
For what it's worth, it *appears* that an upstream change somewhere caused this, you can get the same behavior with chestnut 0.7.0 via `lein new chestnut sample` followed by `lein ancient upgrade` (this would also seem to rule out problems related to hyphen/underscore conversion).

**Comment by timgilbert** [comment]
Looks like the problem was that the order of directories in the call to figwheel-sidecar was backwards (`env` depends on `src`, so `src` needs to come first in the paths vector, it seems). I verified #107 in your sesame-seed template and it seems to work again.

I imagine the actual compiler error (missing ns `sesame-seed/core` when compiling `sesame-seed/main` might be nested inside the exception emitted [here in the clojurescript compiler](https://github.com/clojure/clojurescript/blob/master/src/clj/cljs/compiler.clj#L1122), but it doesn't seem to be printed in the output.

The above fix should take care of #83 as well.

**Comment by plexus** [comment]
Fixed by #107. Thanks @timgilbert!
